.github/workflows/sonar.yaml

```diff
@@ -31,9 +31,14 @@ jobs:
           path: ~/.sonar/cache
           key: ${{ runner.os }}-sonar-${{ steps.date.outputs.date }}
       - uses: actions/cache@v2
+        id: mvn-cache
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ steps.date.outputs.date }}-unitTests
+          key: ${{ runner.os }}-maven-unified-${{ steps.date.outputs.date }}
+      - name: Maven go offline
+        id: mvn-offline
```

Comment:
What does `go-offline` do again?

Reply:
downloads all maven dependencies including plugin etc for your project